src/middleware/validateApiKey.ts

```diff
@@ -79,7 +84,7 @@ export default async (req, res, next) => {
 
     const developer = await findDeveloper(apiKey);
 
-    if (developer) {
+    if (developer && developer.usage) {
```

Comment:
there's no need to check to see if a developer has `usage` on it since we have require developer to always have it - lets revert this line